[Bug] Transition only applying first class

Hugo-Le-Goff opened the pull request:
Hi !

I just discovered this repo and I was trying the dropdown controller. When I tried to change de transitions style, I realised that it was not working anymore and only applying the first class of the list.

So I'm proposing a quick fix for it, maybe there is others controllers to fix. It's late and I don't have time for this right now.

Thanks for your job and I hope my fix proposal is good.

jaywhy commented:
@Hugo-Le-Goff  looks good to me.

You're right. The singular version only returns the first class.

https://stimulus.hotwired.dev/reference/css-classes#properties

Hugo-Le-Goff commented:
I updated the doc for the transition.js, with .: 
- .join(' ') 
- added the "static classes" to be able to access the classes variables
- changed the data classes in the html to end with "-class" which is mandatory to be included as class variable

Also, I'm wondering why this transition feature is natively applied to Dropdown only and not to others components. I don't know if it is relevant for other components, I'm still discovering tailwind, stimulus and this repo.
